### PR TITLE
Table view cell desigh

### PR DESCRIPTION
**Pull request by khrystyna-sb**

### CODE REVIEW

File: Countries/ListViewController.swift [hunk 1]
```diff
@@ -6,15 +6,24 @@
...
     
+    enum TableConstants {
+        static let heightForRow: CGFloat = 179.0
+    }
+
     var countries: [CountriesApiQuery.Data.Country] = []
     
+    let headerView: UIImageView = {
+        var headerView = UIImageView()
+        let image = UIImage(named: "headerList")
```
Comment (Globalika) [inline Countries/ListViewController.swift]: better move "headerlist" to constants

File: Countries/CountryTableViewCell.swift [hunk 2]
```diff
@@ -11,20 +11,41 @@ class CountryTableViewCell: UITableViewCell {
...
+    
+    private let verticalStackView: UIStackView = {
         let stackView = UIStackView()
         stackView.axis = .vertical
         stackView.distribution = .fillEqually
         stackView.translatesAutoresizingMaskIntoConstraints = false
         return stackView
     }()
     
+
```
Comment (Globalika) [inline Countries/CountryTableViewCell.swift]: looks like extra empty line

**Review by Globalika — CHANGES_REQUESTED** [review]

**Review by Globalika — APPROVED** [review]

File: Countries/Assets.xcassets/earth.imageset/Contents.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    },
+    {
+      "idiom" : "universal",
+      "scale" : "3x"
+    }
+  ],
+  "info" : {
+    "author" : "xcode",
+    "version" : 1
+  }
```
Comment (apolwork) [inline Countries/Assets.xcassets/earth.imageset/Contents.json]: Do you plan to use 2x and 3x scale images? If not, adjust your image settings accordingly

File: Countries/Assets.xcassets/headerList.imageset/Contents.json [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    {
+      "idiom" : "universal",
+      "scale" : "3x"
+    }
+  ],
+  "info" : {
+    "author" : "xcode",
+    "version" : 1
+  }
+}
```
Comment (apolwork) [inline Countries/Assets.xcassets/headerList.imageset/Contents.json]: saa

File: Countries/CountryTableViewCell.swift [hunk 5]
```diff
@@ -11,13 +11,33 @@ class CountryTableViewCell: UITableViewCell {
     
     private enum LayoutConstants {
         static let indent: CGFloat = 2.0
-        static let imageHeight: CGFloat = TableConstants.heightForRow - indent
-        static let imageWidth: CGFloat = imageHeight * 1.5
+        static let indentBossFromCell: CGFloat = 15.0
```
Comment (apolwork) [inline Countries/CountryTableViewCell.swift]: Boss?
I would suggest you to adjust your naming.

File: Countries/CountryTableViewCell.swift [hunk 6]
```diff
@@ -11,13 +11,33 @@ class CountryTableViewCell: UITableViewCell {
...
+        static let spacing: CGFloat = 20
+    }
+    
+    private enum ColorConstants {
+        static let gradientFirstColor = CGColor(red: 1, green: 228/255, blue: 133/255, alpha: 0.5)
```
Comment (apolwork) [inline Countries/CountryTableViewCell.swift:19]: Does it return correct color?

File: Countries/ListViewController.swift [hunk 7]
```diff
@@ -55,9 +66,16 @@ class ListViewController: UITableViewController {
     override func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
         return TableConstants.heightForRow
     }
+    
+    override func tableView(_ tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
+        
+        let header = UITableViewHeaderFooterView()
+        header.addSubview(headerView)
+        return header
```
Comment (apolwork) [inline Countries/ListViewController.swift]: Should it have some additional constraints or frame setting?

**Review by apolwork — CHANGES_REQUESTED** [review]

File: Countries/CountryTableViewCell.swift (hunk 6 above)
Comment (khrystyna-sb) [reply Countries/CountryTableViewCell.swift:19]: yes, it does :)

File: Countries/ListViewController.swift (hunk 7 above)
Comment (khrystyna-sb) [reply Countries/ListViewController.swift]: sorry, it should, but I've just changed it in other branch for header

**Review by apolwork — APPROVED** [review]

**Pull request merged**